### PR TITLE
Create TRIAGE_GUIDE.md

### PR DESCRIPTION
**Pull request by bdougie**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Release

## Description
Adding details about the Triage role

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [x] readme
- [ ] no documentation needed

### CODE REVIEW

File: TRIAGE_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Express Triager Guide
```
Comment (yg) [inline TRIAGE_GUIDE.md:1]:
```suggestion
# Open Sauced Triager Guide
```

**Comment by bdougie** [comment]
FYI @Brawrdon @filiptronicek

File: TRIAGE_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Open Sauced Triager Guide
+
+## Issue Triage Process
+
+When a new issue or pull request is opened the issue will be labled with `needs triage`.
```
Comment (yg) [inline TRIAGE_GUIDE.md:5]: Typo

```suggestion
When a new issue or pull request is opened the issue will be labeled with `needs triage`.
```

File: TRIAGE_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+If a traige team member is available they can help make sure all the required information
```
Comment (yg) [inline TRIAGE_GUIDE.md:6]: Typo

```suggestion
If a triage team member is available they can help make sure all the required information
```

File: TRIAGE_GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+is provided. Depending on the issue or PR there are several next labels they can add for further
+classification:
+
+* `needs triage`: This can be kept if the triager is unsure which next steps to take
+* `awaiting more info`: If more info has been requested from the author, apply this label.
+* `question`: User questions which do not appear to be bugs or enahncements.
```
Comment (yg) [inline TRIAGE_GUIDE.md:12]: Typo

```suggestion
* `question`: User questions that do not appear to be bugs or enhancements.
```

File: TRIAGE_GUIDE.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+* `bug`: Issues that present a reasonable conviction there is a reproducible bug.
+* `enhancement`: Issues that are found to be a reasonable candidate feature additions.
+
+In all cases, issues may be closed by maintainers if they don't receive timely response when
+further information is sought, or when additional questions are asked.
+
+## Approaches and Best Practices for getting into triage contributions
+
+Review the organization's [ProjectBoard](https://github.com/open-sauced/open-sauced/projects/1),
+pay special attention to the bugs. This gives you an overview of the current bugs and feature in the repository.
```
Comment (yg) [inline TRIAGE_GUIDE.md:23]: To Plural

```suggestion
pay special attention to the bugs. This gives you an overview of the current bugs and features in the repository.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -60,6 +60,13 @@ We actively welcome your pull requests.
 
 If you plan to contribute a change based on an open issue, please assign your self by commenting on the following word `.take`. Issues that are not assigned are assumed open, and to avoid conflicts, please assign yourself before beginning work on any issues.
```
Comment (NotMoni) [inline CONTRIBUTING.md:61]: `your self` into yourself

File: TRIAGE_GUIDE.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+classification:
+
+* `needs triage`: This can be kept if the triager is unsure which next steps to take
+* `awaiting more info`: If more info has been requested from the author, apply this label.
+* `question`: User questions that do not appear to be bugs or enhancements.
+* `discuss`: Topics for discussion. Might end in an `enhancement` or `question` label.
+* `bug`: Issues that present a reasonable conviction there is a reproducible bug.
+* `enhancement`: Issues that are found to be a reasonable candidate feature additions.
+
+In all cases, issues may be closed by maintainers if they don't receive timely response when
```
Comment (NotMoni) [inline TRIAGE_GUIDE.md:17]: receive **a** timely

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -58,7 +58,14 @@ We actively welcome your pull requests.
 
 ## Issues
 
-If you plan to contribute a change based on an open issue, please assign your self by commenting on the following word `.take`. Issues that are not assigned are assumed open, and to avoid conflicts, please assign yourself before beginning work on any issues.  
+If you plan to contribute a change based on an open issue, please assign yourself by commenting on the following word `.take`. Issues that are not assigned are assumed open, and to avoid conflicts, please assign yourself before beginning work on any issues.  
+
+### Triage Team
+The Triage team is inspired by [expressjs/express](https://github.com/expressjs/express/blob/master/Triager-Guide.md). This team exists to create a path to contributing to this project and open source. All Triage Team members are expected to follow this guide: [TRIAGE_GUIDE.md](/add-triage-doc/TRIAGE_GUIDE.md)
+
+> **There are no minimum requirements for being a member of the team.**
```
Comment (Brawrdon) [inline CONTRIBUTING.md:66]: Maybe change to **There are no minimum requirements to become a member the the Triage Team**. It currently seems like you're saying there's no minimum requirements to **stay** a part of the team when a level of participation is expected, see: 
> - Lack of participation for more than 6 months

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -58,7 +58,14 @@ We actively welcome your pull requests.
 
 ## Issues
 
-If you plan to contribute a change based on an open issue, please assign your self by commenting on the following word `.take`. Issues that are not assigned are assumed open, and to avoid conflicts, please assign yourself before beginning work on any issues.  
+If you plan to contribute a change based on an open issue, please assign yourself by commenting on the following word `.take`. Issues that are not assigned are assumed open, and to avoid conflicts, please assign yourself before beginning work on any issues.  
+
+### Triage Team
+The Triage team is inspired by [expressjs/express](https://github.com/expressjs/express/blob/master/Triager-Guide.md). This team exists to create a path to contributing to this project and open source. All Triage Team members are expected to follow this guide: [TRIAGE_GUIDE.md](/add-triage-doc/TRIAGE_GUIDE.md)
```
Comment (Brawrdon) [inline CONTRIBUTING.md:64]: > This team exists to create a path to contributing to this project and open source.

This team exists to create a path for making contributions to this project and open source.

**Pull request merged**